lib/redis/cluster.rb

```diff
@@ -238,6 +239,29 @@ def try_send(node, method_name, *args, retry_count: 3, &block)
       raise
     end
 
+    def _scan(command, &block)
+      input_cursor = command[1].to_i
+
+      client_index = input_cursor % 2**8
```

Comment:
Probably not much, but we could directly use `256` to not repeatedly recompute that exponent.